src/database/utilities/createRecord.js

```diff
@@ -774,6 +776,38 @@ const createRequisitionItem = (database, requisition, item, dailyUsage, stockOnH
     sortIndex: requisition.items.length + 1,
   });
 
+  if (program) {
+    const { startDate, endDate } = period;
+    const requisitions = database
+      .objects('Requisition')
+      .filtered('otherStoreName == $0 && program != null', otherStoreName)
+      .sorted('period.startDate');
+
+    if (requisitions.length) {
+      const { id: realItemId } = realItem;
+      const matchedItem = requisitions[0].items.find(({ item: { id } }) => id === realItemId);
+      if (matchedItem) {
+        requisitionItem.openingStock = matchedItem.closingStock;
+      }
```

Comment:
Bit finicky but finding the last requisitions closing stock for this requisitions opening stock default value